_topic_map.yml

```diff
@@ -302,8 +302,22 @@ Name: roxctl CLI
 Dir: cli
 Distros: openshift-acs
 Topics:
-- Name: Getting started with the roxctl CLI
-  File: getting-started-cli
+- Name: Installing the roxctl CLI
+  File: installing-the-roxctl-cli
+- Name: Authenticating by using the roxctl CLI
+  File: authenticating-by-using-the-roxctl-cli
```

Comment:
Not sure whether it's better to say "Authenticating roxctl CLI with Central/RHACS" here, any thoughts on how this is best worded?

Reply:
@kcarmichael08 WDYT?

Reply:
So looking at the content here, it actually seems to be about using the roxctl CLI to log in to Central. It actually doesn't even really seem like it belongs in this location because it's documenting a login procedure, not something to do with setting up roxctl. I don't think changing the title works because it currently describes what the actual content is, but maybe it should be moved down in the TOC under "Using the roxctl CLI". The TOC seems to be structured like - 1) what is roxctl; 2) how to use roxctl; 3) things you can do with different roxctl commands. So this would be under that third part.

Reply:
ack